addon/components/nav-route.js

```diff
@@ -34,6 +35,16 @@ export default Component.extend({
     }
   },
 
+  // == Computed Properties ===================================================
+
+  @readOnly
+  @computed('icon')
+  iconHref (icon) {
+    if (icon && icon.includes('/')) {
```

Comment:
Is it possible that a path of `my.svg` could be used?

Reply:
Technically, yes. Do you think it would be better to look for `.svg`?

Reply:
Seems reasonable.  That makes me think of some other questions though.  What happens if the path is not a valid one?  If the svg cannot be found what will the behavior be?  I'm not sure of the interplay with the svg spec and not finding a resource and whether we want to or are able to react to it in a user-friendly way.  Also, can any file extension be used in the path just as long as the response is proper SVG content?

Reply:
It's not really in the spirit of the change, though. I think in that case you would be better off using icon packs. 

We need the URI because we're referencing icons across apps on the same server.

Reply:
If we handle a bad path though visually then we never need to check for specific characters in the icon path because success will be success and failure will be handled as desired so it's only an issue of whether something is provided or not.

Reply:
Does this need to be a computed property, or more specifically, one that actually watches the `icon` property?  Is it expected to be a bound value that is going to/can change?

Reply:
The icon is simply blank if the path is incorrect. This is not at all user-friendly since I'm not sure how one would even test that the path is valid, but I don't see how to improve upon this?

Reply:
Being blank is the UX-way of handling this then (whether ultimately desired or not) so it seems that we only need to check that `icon` has a value and not necessarily any specific value.

Reply:
The reason for the check (at least in my other component) was to know whether the `icon` value should be treated as an external `.svg` file or if it should be treated as the name of an icon in an icon pack. If we aren't inspecting `icon`, then I think the link to an external SVG needs to be provided in a separate property, doesn't it?